openlibrary/templates/home/stats.html

```diff
@@ -1,42 +1,46 @@
 $def with (stats=None)
 $if stats:
-    <div class="head">
+    <div id="home-stats">
+      <div class="head">
         <h2 class="inline">Around the Library</h2>
         <p class="inline sansserif large collapse">&nbsp;Here's what's happened over the last 28 days. More <a href="/recentchanges">recent changes</a>.</p>
-    </div>
+      </div>
```

Comment:
Looks like there's a net addition of +1 open `<div>` tag, but I don't immediately see an added closing tag. Maybe verify tags are balanced. (Oh, HAML, where art thou?!?)

Reply:
It's closed below on line 44